aries-site/src/pages/foundation/typography.mdx

```diff
@@ -63,3 +63,24 @@ For accessibility reasons, always ensure [semantically correct heading levels](h
 >
   <TextSizeExample />
 </Example>
+
+## Fonts for offline usage
+
+In rare cases where a product needs to reference the MetricHPEXS fonts but users
+don't have internet access from their browser or maybe a designer needs the fonts
+locally, you can download the fonts for local use.
+
+MetricHPEXS-Regular [woff2](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff2),
+[woff](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff)
```

Comment:
Designers will want to reference .otf versions. Those files are located in the same directory, just with the .otf extension.
```suggestion
MetricHPEXS-Regular [woff2](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff2),
[woff](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff),
[otf](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.otf),
```